Updated content for Supported Queries (#2382)
couchbase/docs-server: modules/fts/pages/fts-supported-queries-numeric-range.adoc

By default, [.param]`min` is inclusive and [.param]`max` is exclusive.

> A numeric range Query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].

Reviewer: ```suggestion
A demonstration of the numeric range Query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

Author: updated
